tensorflow/python/ops/check_ops.py

```diff
@@ -87,6 +87,260 @@ def _shape_and_dtype_str(tensor):
   """Returns a string containing tensor's shape and dtype."""
   return 'shape=%s dtype=%s' % (tensor.shape, tensor.dtype.name)
 
+def _unary_assert_doc(sym, sym_name):
+  """
+  Common docstring for assert_* ops that evaluate a unary predicate over every 
+  element of a tensor.
+
+  Args:
+    sym: Mathematical symbol for the check performed on each element, i.e.
+      "> 0"
+    sym_name: English-language name for the op described by sym
+  """
+  def _decorator(func):
+    opname = func.__name__
```

Comment:
Lint Error: Missing function docstring [missing-docstring]

Reply:
Added docstring to `_decorator` function.